backend/src/core/models/user/user.ts

```diff
@@ -35,6 +35,8 @@ export class User extends Model<User> {
   })
   email!: string
 
+  // to be accurate, this should be api_key_ash
+  // should I rename this for clarity?
```

Comment:
up to ya but u'll need a migration for this

Reply:
OK think I can do it in a separate PR (with similar refactoring to get rid of all the `!` and stuff)